docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+services:
+
+  db:
+    volumes: 
+    - testvolume:/home/scribe-server/volume
+    image: mariadb
+    environment:
```

Comment:
It could be good to also set here the following environment variables: 
- `MARIADB_USER`
- `MARIADB_PASSWORD`
- `MARIADB_DATABASE`

I believe that would also create a new test database already with credentials within the container when the stack is spun up. We could then test if Scribe-Server is able to connect with the credentials here.